cros_gralloc/cros_gralloc_driver.h

```diff
@@ -55,7 +55,8 @@ class cros_gralloc_driver
 	cros_gralloc_driver operator=(cros_gralloc_driver const &);
 	cros_gralloc_buffer *get_buffer(cros_gralloc_handle_t hnd);
 
-	struct driver *drv_;
+	struct driver *drv_kms;
+	struct driver *drv_render;
```

Comment:
drv_kms should be changed as drv_kms_
drv_render should be changed as drv_render_